skiko/src/commonTest/kotlin/org/jetbrains/skiko/TypefaceTest.kt

```diff
@@ -36,6 +37,8 @@ class TypefaceTest {
 
         assertTrue(inter.getTableData("loca")!!.size > 0)
         assertEquals(2816, inter.unitsPerEm)
+
+        assertCloseEnough(Rect(left=-0.7386364f, top=-1.0909091f, right=2.5830965f, bottom=0.31960228f), inter.bounds)
```

Comment:
Maybe we should test font-related things will less precision, e.g. `epsilon = 0.01f`. It seems to be close enough for real-world cases and will possibly prevent flaky tests.

Reply:
The topic of what would be the most optimal to check in common all the data that actually platform-specific is definitely something we need to thing about it's just that not in this particular PR  - for now all platforms are passing )